src/sentry/models/relay.py

```diff
@@ -22,7 +22,7 @@ class Meta:
         db_table = "sentry_relayusage"
 
 
-@control_silo_with_replication_model
+@region_silo_only_model
```

Comment:
I have no immediate objection to changing this, but let's check on the project's Slack channel whether there's a known reason that it needs to be on the control silo.

Reply:
i got the okay to change this!